factotum/environment.py

```diff
@@ -100,6 +100,11 @@ def GUNICORN_OPTS(cls):
             for k, v in entry.split("=")
         }
 
+    @property
+    def ENABLE_GOOGLE_ANALYTICS(cls):
+        default = False
+        return cls._get("ENABLE_GOOGLE_ANALYTICS", default, prefix=False)
```

Comment:
```suggestion
        return cls._get("ENABLE_GOOGLE_ANALYTICS", default, prefix=False) in cls.truevals
```

This will allow the variable to always normalize to `True`/`False`. Without it, the variable could be set to `"false"`, `"no"`, or `"0"`, but still evaluate to `True` because it would be a non-empty string.